go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/spf13/viper v1.7.0
+	github.com/stretchr/testify v1.6.1
```

Comment:
Not sure how you feel about adding this library in the PR -- saved a bunch of `if diff ... ` statements and seems to have the same functionality